binding/lua/docs/TUTORIAL.md

```diff
@@ -0,0 +1,154 @@
+# Integrate multiverso into torch project
+
+## Setup
+
+1. Install Multiverso shared object by referring to the
+   [Build](../../../README.md#build) instruction of multiverso project.
+1. Install multiverso torch package by referring to the
+   [Installation](../README.md#Installation) instruction of multiverso torch/lua
+   binding.
+
+## Package Initialization
+
+Load and initialize multiverso package and then get some useful parameters at
+the beginning of the whole project.
+
+```lua
+-- Load multiverso.
+local multiverso = require 'multiverso'
+-- Init multiverso.
+multiverso.init()
+-- Get total number of workers.
+multiverso.num_workers = multiverso.num_workers()
+-- Get the id for current worker.
+multiverso.worker_id = multiverso.worker_id()
+-- Easy access to check whether this is master worker.
+multiverso.is_master = multiverso.worker_id == 0
```

Comment:
It should be OK to call it `master worker`. For example, we can only see the log printing from the worker with id 0. It's a great creation!